hudi-flink/src/main/java/org/apache/hudi/sink/partitioner/profile/WriteProfiles.java

```diff
@@ -131,7 +133,7 @@ public static void clean(String path) {
         })
         // filter out crushed files
         .filter(Objects::nonNull)
-        .filter(fileStatus -> fileStatus.getLen() > 0)
+        .filter(fileStatus -> fileStatus.getLen() > MAGIC.length)
         .collect(Collectors.toList());
```

Comment:
What if the file is a log file ?

Reply:
This only filter parquet file that footer not written, log file still filter by fileSize > 0 because we can't predict problems by file size.

Reply:
The committed file can be a log file, what do you mean by `log file still filter by fileSize > 0` ?

Reply:
Sorry, I will fix this.